File: LLM-Translations/GPT-4/avatar/Python/Go/codeforces_32_B.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var s string
	fmt.Scanln(&s)
	result := ""
	i := 0
	for i < len(s) {
		if string(s[i]) == "." {
			result += "0"
		} else {
			i++
			if i < len(s) && string(s[i]) == "." {
				result += "1"
			} else {
				result += "2"
			}
		}
		i++
	}
	fmt.Println(result)
}

// End of Code

File: LLM-Translations/GPT-4/avatar/Python/Java/codeforces_32_B.java
public class codeforces_32_B {
    public static void main(String[] args) {
        java.util.Scanner scanner = new java.util.Scanner(System.in);
        String s = scanner.nextLine();
        StringBuilder result = new StringBuilder();
        int i = 0;
        while (i < s.length()) {
            if (s.charAt(i) == '.') {
                result.append('0');
            } else {
                i++;
                if (s.charAt(i) == '.') {
                    result.append('1');
                } else {
                    result.append('2');
                }
            }
            i++;
        }
        System.out.println(result.toString());
    }
}
// End of Code
